cli/boards.go

```diff
@@ -77,6 +77,39 @@ func UploadBoardDefsAction(ctx *cli.Context) error {
 	return nil
 }
 
+// GetBoardDefsAction is the corresponding action for "board get".
+func GetBoardDefsAction(c *cli.Context) error {
+	orgArg := c.String(organizationFlag)
+	nameArg := c.String(boardFlagName)
+	versionArg := c.String(boardFlagVersion)
+
+	if versionArg == "" {
+		versionArg = "latest"
+	} else if !supportedVersionRegex.MatchString(versionArg) {
+		return fmt.Errorf("invalid version %s. Must use semver 2.0.0 specification for versions", versionArg)
```

Comment:
I liked this check when uploading, but for downloads, shouldn't we allow any version people want? If someone manages to get some non-conforming version number uploaded, I think there should still be a way to download it again. I'd remove this line, but I'm open to pushback.

Reply:
hmmm we have a check in the upload for entering a valid version so I don't see how a non-conforming version would get uploaded, I was thinking w this that if someone didn't know the versioning spec and tried to do like` version=1` they would get this error and then know to do `1.0.0`. versus if we don't have this they would just get a` package not found` error which is less helpful.

Reply:
> I don't see how a non-conforming version would get uploaded

If we decide to change which versions we allow to be uploaded, we'd need to change it in 2 places (both in uploading, and then again in downloading). That's easy to forget: you want to change what versions are allowed to be uploaded, so you change the uploader, and don't even consider the downloader. To avoid this pitfal, IMO, there should be no restrictions on what you can download, so that we don't need to remember this in the future. 

You're right that users need to specify a valid version to download, but hopefully that's easy to figure out. Is there some other (ideally already-built) tool that lists all valid versions of a given package?

Reply:
that's fair, i'll take it out. There is a listPackages API endpoint that can eventually be used to create another command to list all the board def files in the org, we could add that once this is scoped.